libs/project.js

```diff
@@ -17,15 +17,15 @@ module.exports = class Project {
                     reject(error);
             }
             try{
-                fs.copyFileSync(Schema.snack_path +Schema.template + Schema.config_file, Schema.project_path + Schema.config_file);
+                fs.copyFileSync(Schema.gallactic_path +Schema.template + Schema.config_file, Schema.project_path + Schema.config_file);
```

Comment:
It's better we changed it to gnet_path rather than gallactic_path, because it's actually the path of the gnet files.